src/node/interfaces.cpp

```diff
@@ -384,12 +383,13 @@ class NodeImpl : public Node
                     /* verification progress is unused when a header was received */ 0);
             }));
     }
+    NodeContext* m_context{nullptr};
```

Comment:
In commit "refactor: make all NodeImpl and ChainImpl members public" (b9a715ea03506bc6aa647d3df1cd177221d30dd0)

Would encourage putting class member variables all together and not interspersing them with methods. If I'm trying to understand what a class does and what its role is in the application, knowing what data it contains is maybe the single most important piece of information I could have, and it's to see if the data fields aren't declared in a single place.

#21525 messed up the style a bit but the original style and style I'd recommend would be:

```c++
class NodeImpl : public Node
{
public:
  // ... virtual methods first ...
  // ... nonvirtual helper methods next ...
  // ... data fields last ...
};
```

Reply:
Done, and it's a smaller diff too. Thanks for clarifying!